docs/list-style-type.md

```diff
@@ -0,0 +1,57 @@
+> Typography
+
+# List Style Type
+
+Utilities for controlling the bullet/number style of a list.
+
+## Quick reference
+
+<qr-table />
+
+## Basic usage
+### Setting the list style type
+To create bulleted or numeric lists, use the `list-disc` and `list-decimal` utilities.
+
+<container>
+  <div class="flex flex-col gap-8 p-16">
+    <div>
+      <span class="s-text dark:s-text-inverted mb-3">list-disc</span>
+      <ul class="!list-disc !list-inside s-text dark:s-text-inverted !pl-0">
+        <li>An orbit is a regular, repeating path.</li>
+        <li>Orbiting objects, include planets, moons, asteroids, and manmade devices.</li>
+        <li>Objects orbit each other because of gravity.</li>
+      </ul>
+    </div>
+    <div>
+      <span class="s-text dark:s-text-inverted mb-3">list-decimal</span>
+      <ul class="!list-decimal !list-inside s-text dark:s-text-inverted !pl-0">
+        <li>An orbit is a regular, repeating path.</li>
+        <li>Orbiting objects, include planets, moons, asteroids, and manmade devices.</li>
+        <li>Objects orbit each other because of gravity.</li>
+      </ul>
+    </div>
+    <div>
+      <span class="s-text dark:s-text-inverted mb-3">list-none</span>
+      <ul class="!list-none !list-inside s-text dark:s-text-inverted !pl-0">
+        <li>An orbit is a regular, repeating path.</li>
+        <li>Orbiting objects, include planets, moons, asteroids, and manmade devices.</li>
+        <li>Objects orbit each other because of gravity.</li>
+      </ul>
+    </div>
+  </div>
+</container>
+
+```html
+<ul class="list-disc ...">
```

Comment:
We are styling `ul` in resets with `list-style: none`, so as long as Warp users are applying our resets, any `list-style`-controlling classname will need an important rule, like we see in the examples above. Should we mention this on this page, or is it fine that users figure it out on their own?